packages/api/src/command/medical/patient/create-patient.ts

```diff
@@ -44,12 +44,13 @@ export const createPatient = async (patient: PatientCreateCmd): Promise<Patient>
   // Intentionally asynchronous - it takes too long to perform
   cwCommands.patient.create(newPatient, facilityId).catch(processAsyncError(`cw.patient.create`));
 
-  // TODO: NEED CQ PATIENT DIRECTORY SEARCH
-  // const organization = await getOrganizationOrFail({ cxId });
+  const organization = await getOrganizationOrFail({ cxId });
+  const facility = await getFacilityOrFail({ cxId, id: facilityId });
```

Comment:
Lets just create a variable on line 32 so we dont duplicate this call

Reply:
Ah, didn't notice we already checked the facility